src/lsm/groove.zig

```diff
@@ -910,9 +910,7 @@ pub fn GrooveType(
             // Sanity check to ensure the caller didn't accidentally pass in an alias.
             assert(new != old);
 
-            if (has_id) {
-                assert(old.id == new.id);
```

Comment:
:+1: I actually do line one-line without braces form quite a bit more, when it works

Reply:
Yes, we don't need to use kernel style indented returns etc.